webextensions/api/extension.json

```
@@ -406,7 +406,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/extension/setUpdateUrlData",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "10"
```

:x: https://github.com/chromium/chromium/commit/a65882cfd89af9fc51fc34781fd9ebfc4978ea4c in [9.0.581.0](https://github.com/chromium/chromium/releases/tag/9.0.581.0) before 9.0.597.

```suggestion
                "version_added": "9"
```